ctc_lib/container_tool_command_base.go

```diff
@@ -60,12 +63,19 @@ func (ctb *ContainerToolCommandBase) AddCommand(command CLIInterface) {
 
 func (ctb *ContainerToolCommandBase) AddFlags() {
 	// Add template Flag
-	ctb.PersistentFlags().StringVarP(&flags.TemplateString, "template", "t", emptyTemplate, "Output format")
-	ctb.PersistentFlags().VarP(types.NewLogLevel(defaultLogLevel, &flags.LogLevel), "loglevel", "l", "LogLevel")
+	ctb.PersistentFlags().StringVarP(&flags.TemplateString, "template", "t", constants.EmptyTemplate, "Output format")
+	ctb.PersistentFlags().VarP(types.NewLogLevel(constants.DefaultLogLevel, &flags.LogLevel), "logLevel", "l", "LogLevel")
+	ctb.PersistentFlags().BoolVarP(&flags.UpdateCheck, "updateCheck", "u", true, "Run Update Check") // TODO Add Update Check logic
+	viper.BindPFlag("updateCheck", ctb.PersistentFlags().Lookup("updateCheck"))
+	// Also Log to StdOut
+	ctb.PersistentFlags().BoolVar(&flags.AlsoLogToStdOut, "alsoLogToStdOut", false, "Also Log to Std Out")
```

Comment:
I'm not a huge fan of this name. Could we base it off of the log level instead, and make it automatic?

Reply:
Ah. I have seen a lot of python command line tool with that flag.  Its used internally too.
Specifying it with log level is not ideal. We dont want to log everything to StdOut and LogLevel will determine Info, Error, Warn etc.
You need two flags at least one for level and one for specifying the destination. 
How about we change this flag to `--verbose`?

We can add also add a flag called maybe `--logToStdOut` which will skip file logging and only log to std out. This flag would be useful when running our command in CI where logs dont need to be archived.

Reply:
Yeah, it's part of the "glog" tool because it's used internally a lot. --verbosity=debug|info|warn|error sounds good to me.

What I was thinking I guess is that we always log to a file at verbosity=debug.
Then we make this flag control the stderr verbosity.

Reply:
sure. Works.